PENDING.md

```diff
@@ -55,6 +55,10 @@ decoded automatically.
 * [\#3670] CLI support for showing bech32 addresses in Ledger devices
 * [\#3711] Update `tx sign` to use `--from` instead of the deprecated `--name`
 CLI flag.
+* [\#3738] Improve multisig UX:
+  * `gaiacli keys show -o json` now includes constituent pubkeys, respective weights and threshold
+  * `gaiacli keys show --show-multisig` now displays constituent pubkeys, respective weights and threshold
+  * `gaiacli tx sign --validate-signatures` now displays multisig signers with their respective weights
```

Comment:
💪 ++

Reply:
`-o json` implies `--show-multisig` then, doesn't it?

Reply:
yes

Reply:
A user can either 1. `-o json` which will show multi-sig info OR 2. `--show-multisig` which will show multisig info in plaintext (non-JSON).